data/c0000000027o.json

```diff
@@ -148,7 +148,7 @@
                             "type": "paragraph",
                             "blockId": "688cc097-1b1f-8734-fe13-dfbe7297093c",
                             "model": {
-                              "text": "Former embroider Pauline Clayton described the gift as \"lovely\""
+                              "text": "Former embroider Pauline Clayton described the gift as \"lovely\" [placed on the tomb of the unknown warrior](/news/articles/c0000000025o)"
```

Comment:
We shouldn't be updating the production article. There should be examples of a caption with a link in the test data (articles 01-24). If you can't find one, I recommend adding this change to one of those articles.

Reply:
👍